python/reduction/column_reduction_2d_bench.py

```diff
@@ -0,0 +1,93 @@
+# RUN: %PYTHON %s 2>&1 | FileCheck %s
+
+# This file contains small benchmarks with reasonably-sized problem/tiling sizes
+# and codegen options.
+
+from ..core.experts import *
+from ..core.harness import *
+from ..core.transforms import *
+
+from .definitions import *
+
+fun_name = 'column_reduction_2d_on_tensors'
+op_name = 'linalg.generic'
+
+################################################################################
+### Compilation strategies.
+################################################################################
+
+
+class ExperimentalSplitAndFuseFillOpExpert(TransformationList):
+
+  def __init__(self, transforms, **kwargs):
+    t = transforms + [
+        # TODO: bufferization before vectorization triggers
+        # a bufferization issue atm.
+        # This is inefficient wrt VTW/VTR forwarding.
+        #Bufferize()
+    ] + StagedLowerVectorsTransformationList() + [LowerToLLVM()]
+    d = {'transforms': t}
+    kwargs.update(d)
+    TransformationList.__init__(self, **kwargs)
+
+
+def all_experts(problem_sizes: List[int]):
+  return [
+      SingleTilingExpert(
+          fun_name=fun_name,
+          op_name=op_name,
+          # Little trick avoids tiling small dimensions and otherwise tile by 128.
+          sizes=[4, 128] if problem_sizes[1] > 256 else [4],
```

Comment:
Had a bunch of those in the transpose, ideally you'd use something with a good name and put it in utils.
But this is so sall that I don't want to slow you down :)